db/schema.rb

```diff
@@ -0,0 +1,65 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2022_10_11_152945) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "entities", force: :cascade do |t|
+    t.string "name"
+    t.integer "amount", default: 0
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "user_id"
```

Comment:
- [ ] Some aspects of your database schema do not reflect the given ERD diagram. `author_id` should be used instead of `user_id`. It is required in this activity that you do not change column names visible in the ERD diagram. Create a thorough association using _Author_ for _User_ model. This way foreign key `author_id` will be generated. I urge that you follow the example for the same in this previous activity/project: **Blog App - processing data in models**


ERD Diagram:
![image](https://user-images.githubusercontent.com/12745474/195990053-5e016510-08d7-4d5b-b01f-c42120a44149.png)